plgrid/rimrock_bash.py

```diff
@@ -0,0 +1,42 @@
+shieldhit_bash: str = """#!/bin/bash
+#SBATCH --nodes 1
+#SBATCH --ntasks 1
+#SBATCH --time=00:04:59
+#SBATCH -A plgccbmc11-cpu
+
+JOB_ID=$(printenv | grep -Po '(?<=^SLURM_JOBID=)\\w*$')
+SUBMIT_DIR=$(pwd)
+SCRATCH_DIRECTORY=/net/pr2/scratch/people/${{USER}}/${{JOB_ID}}
```

Comment:
```suggestion
SCRATCH_DIRECTORY=${{SCRATCH}}/${{JOB_ID}}
```

If I login to ares I already have a location to SCRATCH directory stored under $SCRATCH

Reply:
I have to check it, because for now it might not be available via rimrock